### DISCUSSION TITLE
Can't boot after install Hyprdots

### DISCUSSION THREAD

**Discussion by kneknew** [prasanthrangan/hyprdots#441] [Q&A]
My pc: i3 7100 + Nvidia Gt740
I installed the script according to the instructions. After installing the script, it automatically accesses Hypr and I can use it normally. However, after restarting, I'm stuck at the **"Loading init ramdisk"** screen, everything is unresponsive, but I can reboot with Ctrl + Alt + Del. I have reinstalled arch and script many times but nothing has changed
I installed Arch using the "archinstall" command, using the default partition layout, Grub boot loader, Minimal Configuration, "Linux" kernel.

 Information about my archinstall, issues:
https://drive.google.com/drive/folders/1IjRRAFya-bOktkzr5HN3w1FWnd5OU-SD?usp=sharing

**Thanks everyone!!!**

**Comment by T-Crypt** [comment]
Your problem is GRUB boatloader. 

Install on UEFI partition and select systemd for boot loader.
